modules/ldsg.js

```
@@ -0,0 +1,157 @@
+      u.ModalActionRow().addComponents([
+        new u.TextInput()
+          .setCustomId("renameText")
+          .setLabel("Rename")
+          .setStyle(Discord.TextInputStyle.Short)
+          .setRequired(false)
+          .setPlaceholder("New title for the forum post"),
+      ]),
+      u.ModalActionRow().addComponents([
+        new u.TextInput()
+          .setCustomId("issueText")
+          .setLabel("Set Issue")
+          .setStyle(Discord.TextInputStyle.Short)
+          .setRequired(false)
+          .setPlaceholder("New issue text")
+      ]),
+      u.ModalActionRow().addComponents([
+        new u.TextInput()
+          .setCustomId("causeText")
+          .setLabel("Set Cause")
+          .setStyle(Discord.TextInputStyle.Short)
+          .setRequired(false)
+          .setPlaceholder("New root cause text")
+      ])
+    ).setCustomId("manageModal").setTitle("Manage Ticket");
+    if (int.customId == "suggestionReply") {
+      await int.showModal(replyModal);
+      const submitted = await int.awaitModalSubmit({ time: 5 * 60 * 1000, dispose: true }).catch(() => {
+        return null;
+      });
+      if (!submitted) return int.channel?.send("I fell asleep waiting for your input...");
+      await submitted.deferUpdate();
+      const reply = submitted.fields.getTextInputValue("replyText");
+      const em = u.embed({ author: int.user })
+          .setTitle("Suggestion Feedback")
+          .setDescription(embed.data.description ?? "")
+          .addFields({ name: "Reply:", value: reply })
+          .setFooter({ text: `-LDSG Team` });
+      const member = int.guild.members.cache.get(suggestion.embeds[0].footer?.text ?? "");
+      if (!member) return int.channel?.send("I could not find that member");
+      try {
+        member.send({ embeds: [em] });
+        return int.channel?.send({ content: "Replied to user:", embeds: [em] });
+      } catch (e) {
+        u.errorHandler(e, int);
```

no need to report the error